stories/css-prop-static-object.tsx

```diff
@@ -23,7 +23,6 @@ export const ObjectLiteralSpreadFromFunc = (): JSX.Element => {
       css={{
         display: 'flex',
         fontSize: '50px',
-        color: 'blue',
```

Comment:
Why are we removing this?

Reply:
It's a type error :-) "color is defined multiple times and this won't be used"